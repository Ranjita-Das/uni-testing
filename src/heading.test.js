import React from 'react'
import Heading from './heading.js'
import renderer from 'react-test-renderer'

it('renders heading correctly',()=>{
  const tree2= renderer.create(<Heading />);
  expect(tree2).toMatchSnapshot();
});
