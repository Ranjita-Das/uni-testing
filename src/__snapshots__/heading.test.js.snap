// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders heading correctly 1`] = `
<div>
  <div>
    This is the heading
  </div>
  <div>
    50
  </div>
  <div>
    80
  </div>
  <div>
    100
  </div>
  <div>
    150
  </div>
</div>
`;
